CairoMakie/src/cairo-extension.jl

```diff
@@ -74,3 +74,10 @@ function get_render_type(surface::Cairo.CairoSurface)
     typ == Cairo.CAIRO_SURFACE_TYPE_IMAGE && return IMAGE
     return IMAGE # By default assume that the render type is IMAGE
 end
+
+function restrict_pdf_version!(surface::Cairo.CairoSurface, v::Integer)
```

Comment:
would be simpler to type the `v` with the enum here, then you don't need yet another check. I'd guess you don't even have to manually convert it to Int32 in the ccall

Reply:
I would like to move `restrict_pdf_version!` to Cairo.jl eventually, where they stick to numerical constants instead of enums, so I'd prefer to keep the argument as `Integer`.

You are right that the manual conversion in the ccall is not needed. I removed it.